wrapper/src/nibabies_wrapper/__main__.py

```diff
@@ -735,4 +738,6 @@ def main():
 
 
 if __name__ == "__main__":
+    if '__main__.py' in sys.argv[0]:
+        sys.argv[0] = '%s -m %s' % (sys.executable, __name__)
```

Comment:
We know `__name__ == "__main__"`. We want the `__name__` of the parent module.

```suggestion
    if '__main__.py' in sys.argv[0]:
        from . import __name__ as module
        sys.argv[0] = '%s -m %s' % (sys.executable, module)
```